03.windows/04.solving-problems/03.adguard-logs/docs.en.md

```diff
@@ -8,32 +8,19 @@ visible: true
 
 For analysis and diagnostic of different problems support service might need AdGuard log files. These files contain records about all errors occurred with the software recently. Below is the brief instruction describing the way to get the log files and send them to support service.
 
-1. Open the path with AdGuard log files.
+1. Open the AdGuard settings. Go to the General Settings section, scroll down to the end of the screen and select **Debug** Logging Level.
 
-For Windows 10 click the Search icon on the Taskbar:
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/Windows7.1/logsen.png" />
 
-Type **%programdata%\Adguard** and press Enter.
+2. Reproduce your issue. 
+- It is advisable to specify the reproducing time, it will help our support team to solve this situation.
 
-![](win10_search_EN.png)
+3. Collect the received logs using the **Export Log** button.
 
-If you use Windows 8 or Windows 8.1 – go to the **Search**:
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/Windows7.1/logsen2.png" />
 
-![](side%20menu.png)
+- Save the archive to any folder convenient for you.
 
-Type in **%programdata%\Adguard** and press Enter:
+4. **Important**: return **Standard** logging level. Because **Debug** logging level can slow down the application filtering.
```

Comment:
`4. **Important**: set the logging level back to **Standard**. **Debug** logging level will slow down the application.`